srcpkgs/common-lisp-clx/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'common-lisp-clx'
+pkgname=common-lisp-clx
+version=0.7.5
+revision=1
+build_style=fetch
```

Comment:
Remove build_style variable. Then tarball is extracted by convention, so you don't need `tar -xf`, `cd`, and tar in hostmakedepends.